src/schemas/index.schema.json

```diff
@@ -26,6 +26,11 @@
       "format": "uri-template",
       "description": "The source document to retrieve values from. Known variables in the URI Template are: `repo`, `ref`, `owner`, `path`"
     },
+    "target": {
+      "type": "string",
+      "format": "uri",
+      "description": "The data sink to store the extracted data in. This can be an Excel or Google spreadsheet"
```

Comment:
or azure index, or algolia index.....

Reply:
```suggestion
      "description": "The data sink to store the extracted data in. This can be an Excel or Google spreadsheet, an Azure Index or Algolia index"
```